content/200-concepts/100-components/02-prisma-client/090-raw-database-access.mdx

```diff
@@ -397,6 +397,49 @@ generator client {
 }
 ```
 
+### <inlinecode>$runCommandRaw</inlinecode>
+
+`$runCommandRaw` runs a raw MongoDB command against the database. For example, this query inserts two records with the same `_id`, bypassing normal document validation:
+
+```ts no-lines
+prisma.$runCommandRaw({
+  insert: 'Pets',
+  bypassDocumentValidation: true,
```

Comment:
I don't think we want to promote this option `bypassDocumentValidation`. If MongoDB users are using JSON schema validation, that's good for us since it keeps the data consistent.

Reply:
I'd suggest just removing it and not talking about it in the paragraph above.

Reply:
In that case we'll need a different example, right? I think Dom's idea was to provide a query that isn't something the Prisma client would allow you to do, as a use case for raw queries. So `bypassDocumentValidation` is included to allow two entries with the same id.

Happy to put in something else if you have suggestions?

Reply:
Ah, I didn't realize this property was necessary for the example to work.

Still, I think we can come up with a better example. Untested, but I'd suggest we document the [findAndModify](https://docs.mongodb.com/manual/reference/method/db.collection.findAndModify/#mongodb-method-db.collection.findAndModify) command. Prisma's upsert is not nearly as powerful.


Untested, but something like:
```ts
prisma.$runCommandRaw({
  findAndModify: 'Pets',
  query: { age: { $gt: 10 } },
  update: { type: "Old Cat" }
})
```

Let me know if you want me to test that the code above works as expected. I'll be a bit held up until after the release, but can check it after. In the meantime, feel free to merge this as-is and see about improving the example after.

Reply:
Thanks! I'll leave the current one for the release, and we can improve this later.